lookbook/docs/patterns/12-tables.md.erb

```diff
@@ -33,6 +33,51 @@ end
 
 <%= embed OpPrimer::BorderBoxTableComponentPreview, :default, panels: %i[source] %>
 
+
+
+### Border Box Table specific options
+
+#### Mobile behavior
+
+On mobile, the BorderBoxTable does not scroll, but collapse into two columns (content columns and actions). You can control which columns are shown using the `mobile_columns` method.
+
+As the columns are collapsed, some information might need an additional label, which you can prepend with `mobile_labels`. 
+
+```ruby
+# On desktop, show these columns
+columns :title, :users, :created_at
+
+# On mobile, only two columns are visible
+mobile_columns :title, :users
+
+# For users, show I18n.t(:label_user_plural)
+mobile_labels users: :label_user_plural
```

Comment:
Do we really want to allow a custom label here? In the specification it was written `a flag to show the column header as label`